README.md

```diff
@@ -1,2 +1,135 @@
 # lvl_is_flite
 Voice generation for Icelandic Flite voices
+
+## Preparation
+This section describes where the data for this voice comes from and how it is preprocessed.
+
+### Voice data
+This repo is intended for use with data from [the Talrómur corpus](https://repository.clarin.is/repository/xmlui/handle/20.500.12537/104), and in particular the first voice; lvl_is_f is created using voice f, named Álfur therein.
+There are 2 primary voice-specific files in this voice build. They are, relative to the voice directory: 
+- `etc/txt.done.data`, which contains normalized prompts with inserted phrasing tags
+- `festvox/lexicon.scm` which contains a mix of manual and automatic g2p transcriptions of all words that occur in the source data.
+
```

Comment:
could you add a reference to the paper that stands behind the voice generation ? https://aclanthology.org/2020.sltu-1.44.pdf